example/src/useLazyFetch/Basic.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { useLazyFetch } from 'use-fetch-hooks';
+import { useLazyFetch } from '../../../src';
```

Comment:
This is uglier but it's easier than having to alias `use-fetch-hooks` in both `webpack` and `eslint` config (as well as other tooling).